src/compiler/emitter.ts

```diff
@@ -6314,6 +6461,37 @@ var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, Promi
                 write("}"); // execute
             }
             
+            function emitExportStarFunction(localNames: string): string {
+                const exportStarFunction = makeUniqueName("exportStar");
+
+                writeLine();
+
+                // define an export star helper function
+                write(`function ${exportStarFunction}(m) {`);
+                increaseIndent();
+                writeLine();
+                write(`var exports = {};`);
+                writeLine();
+                write(`for(var n in m) {`);

```

Comment:
I actually just moved this from an inner scope so I could reuse it - changing this is going to change some baselines. :3